web/src/app/components/login-page/login-page.component.ts

```diff
@@ -52,16 +55,30 @@ export class LoginPageComponent implements OnInit {
   saveUser(user: User, provider: string) {
     this.userService.saveUser(user, provider)
       .subscribe((userData) => {
+        this.saveUserSocialRecord();
         this.saveUserSocialStats(userData, provider);
```

Comment:
These do not match the style above

Reply:
Yes, let's follow one style. For now, you could chain the subscribe and follow everywhere

Reply:
Yep lets be consistent 👍 . Async calls should be in the `pipe` and only the final action should be the subscribe with no async

Reply:
okay @eddiejaoude . Will change everywhere and make it consistent.